lib/Confirmed_Ride.dart

```diff
@@ -0,0 +1,349 @@
+                    SizedBox(height: MediaQuery.of(context).size.height/8),
+                  ],
+                )
+              ],
+            ),
+            EditRide()
+          ],
+        )
+    );
+  }
+}
+
+class Header extends StatelessWidget {
+
+  const Header({Key key, this.header}) : super(key: key);
+  final String header;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        Container(
+          margin: EdgeInsets.only(left: 15, top: 15, bottom: 15),
+          child: Text(header,
+              style: TextStyle(
+                  color: Colors.black,
+                  fontSize: 30,
+                  fontFamily: 'SFPro',
+                  fontWeight: FontWeight.bold)),
+        )
+      ],
+    );
+  }
+}
+
+class SubHeader extends StatelessWidget {
+
+  const SubHeader({Key key, this.subHeader1, this.subHeader2}) : super(key: key);
+
+  final String subHeader1;
+  final String subHeader2;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.only(left: 15),
+      child: Row(
+        children: <Widget>[
+          Text(subHeader1, style: TextStyle(color: Colors.grey[800],
+              fontSize: 20,
+              fontFamily: 'SFPro',
+              fontWeight: FontWeight.bold)),
+          SizedBox(width: 10.0),
+          Text(subHeader2,
+              style: TextStyle(color: Colors.green[500], fontSize: 11))
+        ],
+      ),
+    );
+  }
+}
+
+class CustomDivider extends StatelessWidget {
+
+  const CustomDivider({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: MediaQuery.of(context).size.width,
+      height: 4,
+      color: Colors.grey[200],
+    );
+  }
+}
+
+class Contact extends StatelessWidget {
+
+  const Contact({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.only(left: 15),
+      child: Row(
+        children: <Widget>[
+          Icon(Icons.account_circle, size: 50),
+          SizedBox(width: 15),
+          Container(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Text('Davea Butler', style: TextStyle(color: Colors.black,
+                    fontSize: 16,
+                    fontFamily: 'SFPro')),
+                SizedBox(height: 5),
+                Container(
+                  child: Row(
+                    children: <Widget>[
+                      Icon(Icons.phone, size: 15),
+                      SizedBox(width: 7),
+                      GestureDetector(
+                        onTap: () => UrlLauncher.launch("tel://13232315234"),
+                        child: Text('+1 323-231-5234', style: TextStyle(color: Colors.grey[600])),
+                      )
```

Comment:
Could you remove the phone icon and the text showing the number from the UI and instead add a phone icon like in the "Current Ride" prototype in the product flow? https://www.figma.com/file/jFRQPUXBWS9r3dTRKC04Wd/Carriage-Product-Flow?node-id=0%3A1


This way we don't have to display the driver's number in-app, we can just have a big phone icon that you can click to call.

Reply:
I haven't fully reviewed this yet. Once it looks okay, maybe request a review from Helen or Bryan so that people can start getting their credit for reviews. Then once it is reviewed, you will be the one who gets to merge it in, not the reviewer.

Reply:
I fixed it (removed the text and added a large phone icon)